app/javascript/components/BotChannel.jsx

```diff
@@ -64,7 +64,7 @@ class BotChannelComponent extends Component {
               the <a href="https://developers.facebook.com/docs/messenger-platform/getting-started/app-setup" target="_blank" className="hrefLink">subscribe bot page</a>.
             </Headline>
 
-            <Field label="Callback URL" defaultValue={`${location.protocol}//${location.host}/callback/facebook/`} readOnly />
+            <Field label="Callback URL" defaultValue={`${location.protocol}//${location.host}/callback/facebook/${bot.uuid}`} readOnly />
```

Comment:
I wasn't able to test the generated url (i.e: http://ui.aidaui.lvh.me/callback/facebook/70bd540c-958c-4334-86dd-9aa222f364ea) directly. I've tested it for aida (backend) and the routes (both the old and the new ones) are working properly. I've found some related configuration in the routes.rb file, targeting OmniauthCallbacksController, but I think it's not necessary to make changes there. I think with this changes we are good, but I'm not completely sure.